libs/features/mas/docs/styles.css

```diff
@@ -71,11 +71,14 @@ span.placeholder-failed::after {
 }
 
 merch-card {
+  display: flex;
```

Comment:
imo we shouldn't have styles from merch-card in ccd.html (it's ok to have it on rest of pages)
reason is we have less change to catch an issue since ccd.html deviates from ccd usage by introducing additional styles. Example, in december, we had a space bug between prices that was only discovered on CCD build because in gallery it was 'fixed' by /docs/styles.css

I think on all gallery pages we shouldn't include <link rel="stylesheet" href="styles.css"> at all.

Reply:
Good point @3ch023, let me remember why I have done this and find an alternative.